Ahmadreza-tahmasebi/c6.js

```diff
@@ -0,0 +1,33 @@
+"use strict"
+
+function oneSidePyramidMaker(num){
+    for(var i = 1 ; i <= num ; i++){
+        for(var j = 1 ; j <= (i * 2) - 1 ; j++){
+            process.stdout.write("*");
+        }
+        console.log();
```

Comment:
This is an empty `console.log`

Reply:
Yes, this is used in order to go to the next line in the console.